test/harmonytest.js

```diff
@@ -18524,9 +18524,9 @@ var harmonyTestFixture = {
         },
 
         'func(...a, b)': {
-            index: 9,
+            index: 10,
             lineNumber: 1,
-            column: 10,
+            column: 11,
```

Comment:
Yea, I guess just leave it as is. I don't know what the right thing is. This is where people like @jeffko come in.

What does this mean for `func(...a,)`? That one isn't an error right?

Anyway, tests :) Travis is failing now because you don't have any and it expects 100% coverage (run locally with npm test)

And @DmitrySoshnikov feel free to accept when tests pass.